experimental/framework/get-container/src/getContainer.ts

```diff
@@ -15,6 +13,14 @@ import {
 export interface IGetContainerService {
```

Comment:
We shouldn't need this interface anymore.

Reply:
Yes, I was maintaining this before to keep it compatible with the existing getContainerServices code. Now that tinylicious-client does not have any dependencies on get-container, it no longer uses this interface